services/repository/files/update.go

```diff
@@ -372,6 +372,7 @@ func (err ErrSHAOrCommitIDNotProvided) Error() string {
 
 // handles the check for various issues for ChangeRepoFiles
 func handleCheckErrors(file *ChangeRepoFile, commit *git.Commit, opts *ChangeRepoFilesOptions) error {
+	// FIXME: should it also handle "rename" errors?
```

Comment:
@bytedream here and the FIXME below, should it also handle "rename" errors?

Reply:
I resolved the FIXME by my understanding https://github.com/go-gitea/gitea/pull/34744/commits/33b3e26431bd9edabacc3168b63e70607e302516